src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -49,8 +50,44 @@ void prepare_images(const image_type& image,
                  [](unsigned char val) { return val / 255.f; });
 }
 
+std::vector<std::pair<flex_int, flex_image>> process_output(
+    const shared_float_array& contents, size_t index, size_t batch_size,
+    size_t width, size_t height) {
+  constexpr size_t channels = 3;
+
+  std::vector<std::pair<flex_int, flex_image>> result;
+  result.reserve(batch_size);
+
+  size_t image_size = height * width * channels;
+
+  ASSERT_TRUE(contents.size() == image_size * batch_size);
+
+  const float* start_ptr = contents.data();
+
+  for (size_t index = 0; index < batch_size; index++) {
+    std::vector<uint8_t> image_data;
+    image_data.reserve(image_size);
+
+    size_t start_offset = image_size * index;
+    size_t end_offset = start_offset + image_size;
+
+    std::transform(start_ptr + start_offset, start_ptr + end_offset,
+                   std::back_inserter(image_data), [](float val) {
+                     return (uint8_t)(std::round(val * 255.f));
```

Comment:
Nit: use `static_cast` instead of C-style cast

Also, would be wise to clamp the result before casting to the range [0.f, 255.f]

If we had C++17, you could use `std::clamp` https://en.cppreference.com/w/cpp/algorithm/clamp

For now, I would just re-implement that:
```
float clamp(float v, float low, float high) {
  return (v < low) ? low : (high < v) ? high : v;
}
```